app/src/main/java/com/skyyo/samples/features/userInteractionTrackingResult/UserInteractionTrackingResultViewModel.kt

```diff
@@ -46,11 +45,16 @@ class UserInteractionTrackingResultViewModel @Inject constructor(
         }
     }
 
-    fun goHome() {
-        navigationDispatcher.emit { it.navigate(Destination.SampleContainer.route) }
+    fun onContinueClick() {
+        userIdlingSessionEventDispatcher.startSession()
+        goBack()
```

Comment:
let's use directly 
`navigationDispatcher.emit { it.popBackStack() }`
without private fun

Reply:
Why? I separated starting session and navigation as we usually do

Reply:
But here I don't see any sense of creation goBack() fun since we use it only in one place
(and the block inside is a single line)